packages/bundle/.babelrc

```diff
@@ -7,7 +7,8 @@
     }
   },
   "plugins": [
-    "@babel/proposal-object-rest-spread",
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-transform-runtime",
```

Comment:
We don't have any `async`/`await` code in the bundle package (we do have them in core and UI package).

We need `plugin-transform-runtime`, otherwise, we will be required to use `babel-polyfill`. However, `babel-polyfill` is not designed for components, it is designed to use in the end product, i.e. web site (limited to one `babel-polyfill` for whole solution).

`plugin-transform-runtime` will help us to transform calls to `babel-polyfill` into calls to `babel-runtime`, which is a polyfill package designed for components. `babel-runtime` is localized and don't collide with other polyfills in other packages.